.github/workflows/release.yaml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  release:
+    types: [published]
```

Comment:
So basically, we release to PyPI once the release on GH is created?

Reply:
exactly